scripts/install.sh

```diff
@@ -295,7 +295,7 @@ function main() {
     apply_bash_flag_overrides "$@"
     parse_yaml_into_bash_variables
     parse_kubernetes_target_version
-    discover
+    discover full-cluster
```

Comment:
when is it not "full-cluster"?

Reply:
in join.sh and upgrade.sh top level scripts